src/navigation/Nav.jsx

```diff
@@ -0,0 +1,284 @@
+import scriptLogo from '../../assets/svg/logo--script.svg';
+import withMatchMedia from '../utils/components/withMatchMedia';
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import Icon from '../media/Icon';
+import AvatarMember from '../media/AvatarMember';
+
+import NavItem from './components/NavItem';
+import ProfileDropdown from './components/profile/ProfileDropdown';
+import NotificationsDropdown from './components/notifications/NotificationsDropdown';
+
+const CLASS_UNAUTH_ITEM = 'navItem--unauthenticated';
+const CLASS_AUTH_ITEM = 'navItem--authenticated';
+
+export const DropdownLoader = ({ label }) => (
+	<Section className="valignChildren--center align--center" aria-live="polite">
+		<p className="visibility--a11yHide">{label}</p>
+		<Icon shape="updates" size="l" />
+	</Section>
+);
+
+/**
+ * @param {Object} props component properties
+ * @returns {React.element} Navigation Bar
+ */
+export class Nav extends React.Component {
+	/**
+	 * @return {React.element} the navbar component
+	 */
+	render() {
+		const {
+			media,
+			self,
+			logoAccessible,
+			navItems,
+			dropdownLoaderLabel,
+			localeCode,
+		} = this.props;
+		const {
+			login,
+			signup,
+			proDashboard,
+			explore,
+			messages,
+			notifications,
+			groups,
+			profile,
+		} = navItems;
+		const isLoggedOut = self.status === 'prereg' || !self.name;
+
+		const showScriptLogo = Boolean(media.isAtLargeUp || isLoggedOut);
+		const showSwarmLogo = Boolean(
+			media.isAtMediumUp && !media.isAtLargeUp && !isLoggedOut
+		);
+		const notificationContent = notifications.list ? (
+			<NotificationsDropdown
+				self={self}
+				notifications={notifications.list}
+				onMarkReadAction={notifications.notificationsDropdown.markRead}
+				localeCode={localeCode}
+				emptyContentLabel={notifications.notificationsDropdown.emptyContentLabel}
+				notificationsTitleLabel={notifications.label}
+				generateClassicUrl={
+					notifications.notificationsDropdown.generateClassicUrl
+				}
+			/>
+		) : (
+			<DropdownLoader label={dropdownLoaderLabel} />
+		);
+
+		const profileContent = groups ? (
+			<ProfileDropdown
+				settings={profile.profileDropdown.settings}
+				help={profile.profileDropdown.help}
+				logout={profile.profileDropdown.logout}
+				groupHome={profile.profileDropdown.groupHome}
+				allGroupsLabel={profile.profileDropdown.allGroupsLabel}
+				profile={profile}
+				groups={groups.list}
+			/>
+		) : (
+			<DropdownLoader label={dropdownLoaderLabel} />
+		);
+
+		let unauthItems = [
+			{
+				shrink: true,
+				linkTo: login.link,
+				label: login.label,
+				className: `${CLASS_UNAUTH_ITEM} navItem--login`,
+			},
+			{
+				shrink: true,
+				onAction: () => {},
+				label: signup.label,
+				className: CLASS_UNAUTH_ITEM,
+			},
+		];
+
+		let authItems = [
+			self.isProMember && {
+				shrink: true,
+				linkTo: proDashboard.link,
+				label: proDashboard.label,
+				className: cx(
```

Comment:
you don't need to use `cx` for this case since you don't have any conditional logic. you can just pass the string